src/react-apps/applications/runtime/src/features/form/data/actions/index.ts

```diff
@@ -16,7 +16,7 @@ export interface IFormDataActions extends ActionCreatorsMapObject {
   fetchFormDataFulfilled: (formData: any) => FetchFormData.IFetchFormDataFulfilled;
   fetchFormDataRejected: (error: Error) => FetchFormData.IFetchFormDataRejected;
   completeAndSendInForm: (url: string) => CompleteForm.ICompleteAndSendInForm;
-  completeAndSendInFormFulfilled: () => Action;
+  completeAndSendInFormFulfilled: (reponse: any) => Action;
```

Comment:
Interface on response?

Reply:
Created interface for the instance in several places. Including this.